.github/workflows/macos.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       fail-fast: false
       matrix:
         configuration: [Debug, Release]
-        architecture: [x64]
+        architecture: [x64, arm64]
```

Comment:
Are you sure about this one? I believe there should be no changes in macos.yml, otherwise it will just spawn more workers for no real benefit.